frontend/src/theme/components/Tabs.ts

```diff
@@ -0,0 +1,134 @@
+import {
+  ChakraTheme,
+  ComponentMultiStyleConfig,
+  SystemStyleObjectRecord,
+  ThemingPropsThunk,
+} from '@chakra-ui/react'
+import { getColor } from '@chakra-ui/theme-tools'
+import merge from 'lodash/merge'
+
+export const DRAG_SCROLL_SPEED = 0.8
+
+const variantLine: ThemingPropsThunk<SystemStyleObjectRecord, ChakraTheme> =
+  () => {
+    return {
+      root: {
+        w: '100%',
+      },
+      tablist: {
+        w: '100%',
+        overflowX: 'scroll',
+        borderColor: 'transparent',
+        /* Hide scrollbar for IE and Edge */
+        '&::MsOverflowStyle': 'none',
```

Comment:
I think it is ok to show the scrollbar (since it will only show on scroll)

Reply:
i changed `overflowX` to `auto` but the scrollbar is shown as long as there is overflow. Is that okay?

Reply:
I think that's fine. Clarify with pearly if unclear!

Reply:
I've styled the custom scrollbar! But customisation is limited on Firefox browsers so I just did the best I can. Also, I discussed the padding with pearly and we've decided to use 24px padding so the total height is (24 * 2 + 24 + 4)px = 76px